File: C/queue/qwen-qwq-32b.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

typedef struct {
    int *array;
    int front;
    int rear;
    int capacity;
} Queue;

Queue* createQueue(int capacity) {
    Queue *q = (Queue*)malloc(sizeof(Queue));
    q->capacity = capacity;
    q->front = -1;
    q->rear = -1;
    q->array = (int*)malloc(capacity * sizeof(int));
    return q;
}

void enqueue(Queue *q, int item) {
    if ((q->rear + 1) % q->capacity == q->front) {
        return; // Queue full
    }
    q->rear = (q->rear + 1) % q->capacity;
    q->array[q->rear] = item;
    if (q->front == -1) {
        q->front = q->rear;
    }
}

int dequeue(Queue *q) {
    if (q->front == -1) {
        return -1; // Queue empty
    }
    int item = q->array[q->front];
    if (q->front == q->rear) {
        q->front = -1;
        q->rear = -1;
    } else {
        q->front = (q->front + 1) % q->capacity;
    }
    return item;
}

void freeQueue(Queue *q) {
    free(q->array);
    free(q);
}

int main() {
    FILE *file = fopen("../../inputs.txt", "r");
    if (!file) {
        return 1;
    }

    Queue *q = createQueue(1000); 
    if (!q) {
        fclose(file);
        return 1;
    }

    int num, count = 0;
    while (count < 1000 && fscanf(file, "%d", &num) == 1) {
        enqueue(q, num);
        count++;
    }
    fclose(file);

    free(q); 
    return 0;
}

File: Python/queue/qwen-qwq-32b.py
import random
from collections import deque

class Queue:
    def __init__(self):
        self.container = deque()

    def enqueue(self, item):
        self.container.append(item)

    def dequeue(self):
        return self.container.popleft() if not self.is_empty() else None

    def is_empty(self):
        return len(self.container) == 0

    def size(self):
        return len(self.container)

def main():
    input_path = "inputs.txt"
    
    q = Queue()
    
    try:
        with open(input_path, 'r') as f:
            for line in f:
                line = line.strip()
                if line:
                    value = int(line)
                    q.enqueue(value)
    except FileNotFoundError:
        return

    while not q.is_empty():
        print(q.dequeue())

if __name__ == "__main__":
    main()
